aclocal.m4

```diff
@@ -105,55 +105,21 @@ AC_DEFUN([MP_TOOL_PATH], [dnl
 		[])dnl
 ])
 
-dnl Configure a project contained in a .tar.gz, .tgz or .tar.bz2 tarball,
-dnl extracting it previously, if necessary. Different from AC_CONFIG_SUBDIRS
-dnl (on which this macro is based), you can pass parameters to the
-dnl sub-configure script.
+dnl Configure a project contained in a subdirectory. Different from
+dnl AC_CONFIG_SUBDIRS (on which this macro is based), you can pass parameters
+dnl to the sub-configure script.
 dnl
 dnl Parameters:
-dnl  - The relative path to the tarball
-dnl  - The relative path to the directory that will be extracted from the
-dnl    tarball and contains the configure script to be run (can be a sub directory of the extracted directory)
+dnl  - The relative path to the directory that contains the configure script
+dnl    to be run
 dnl  - Parameters to pass to the configure script
 dnl
-dnl MP_CONFIG_TARBALL([path-to-tarball], [dir-extracted-from-tarball-with-configure], [configure-parameters])
-AC_DEFUN([MP_CONFIG_TARBALL], [
-	dnl Warning: Don't use $GZIP and $BZIP2 here, both tools interpret these
-	dnl env variables as additional parameters.
-	AC_PATH_PROG(GZIP_BIN, [gzip], [])
-	AC_PATH_PROG(BZIP2_BIN, [bzip2], [])
+dnl MP_CONFIG_SUBDIR([path-to-directory], [configure-parameters])
+AC_DEFUN([MP_CONFIG_SUBDIR], [
 
-	mp_tarball="$1"
-	ac_dir=$2
+	ac_dir="$1"
```

Comment:
I think we should remove the quotes here, this becomes `ac_dir=""$1""` in the generated configure file, which I don't think I did on purpose. Same for the copies of this.